docs/stage2/wireguard.md

```
@@ -0,0 +1,105 @@
+
+**3. Read the peer configs out of the pod logs.** `LOG_CONFS=true` makes the image print each peer's configuration and a scannable QR code on startup:
+
+```bash
+kubectl -n vpn logs deployment/wireguard
+```
+
+## Gotchas
+
+!!! danger "The config PVC is created but never mounted, so peer keys do not survive a restart"
+
+    `stage2/vpn/wireguard.tf` declares the `wireguard-config` volume on the pod and binds the PVC, but the container has only one `volume_mount`, for `/lib/modules`. Nothing mounts the claim at `/config`.
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)